drivers/timer/nrf_rtc_timer.c

```diff
@@ -49,7 +49,7 @@ static u32_t counter(void)
  * it by pointer at runtime, maybe?) so we don't have this leaky
  * symbol.
  */
-void rtc1_nrf5_isr(void *arg)
+void rtc1_nrf_isr(void *arg)
```

Comment:
could this function be declared as static?

Reply:
I guess not. Because the arm_irq_vector_table [apparently uses it](https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/kernel/arm_irq_vector_table/src/arm_irq_vector_table.c#L123). I need to rename it in this test as well.

Reply:
Done (renaming of the function in the test).